src/interpret.jl

```diff
@@ -99,7 +99,7 @@ function lookup_or_eval(@nospecialize(recurse), frame, @nospecialize(node))
             elseif f === Base.getproperty
                 return Base.getproperty(ex.args[2], ex.args[3])
             else
-                error("unknown call f ", f)
+                Base.invokelatest(error, "unknown call f ", f)
```

Comment:
If we are on Julia 1.8 and higher, we can do something like this alternatively:
```suggestion
                error(lazy"unknown call f $f")
```